src/sockperf.cpp

```diff
@@ -1527,8 +1527,8 @@ static int proc_mode_server(int id, int argc, const char **argv) {
         { OPT_RXFILTERCB,
           AOPT_NOARG,
           aopt_set_literal(0),
-          aopt_set_string("rxfiltercb", "vmarxfiltercb"),
-          "Use VMA's receive path message filter callback API (See VMA's readme)." },
+          aopt_set_string("rxfiltercb"),
+          "Use extra API receive path message filter callback API (See VMA/XLIO readme)." },
```

Comment:
could you update `OPT_RATE_LIMIT` option with `VMA/XLIO` and `VMA_RING_ALLOCATION_LOGIC_TX/XLIO_RING_ALLOCATION_LOGIC_TX` changes

Reply:
Done